<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemPic</class>
 <widget class="QWidget" name="ItemPic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="wBg" native="true">
     <property name="styleSheet">
      <string notr="true">

QWidget#wBg{
background-color: rgb(255, 255, 255);
border-radius:5px;
color: rgb(41,41,41);
border: 1px solid #ffffff; 
 padding: 5px;}
</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="1" column="0" colspan="2">
       <widget class="QWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #f3f3f3;
</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btnUpload">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>上傳</string>
           </property>
           <property name="icon">
            <iconset resource="../AdbClient/res/res.qrc">
             <normaloff>:/imageUpload.png</normaloff>:/imageUpload.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLbPicClear">
           <property name="text">
            <string>清除</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnChip">
           <property name="styleSheet">
            <string notr="true">
</string>
           </property>
           <property name="text">
            <string>裁切</string>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/capture.png</normalon>
            </iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="lbPic">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/image0.png);
background-color: #f3f3f3;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../AdbClient/res/res.qrc"/>
 </resources>
 <connections/>
</ui>
